.github/workflows/build.yml

```diff
@@ -23,19 +23,19 @@ jobs:
       - run: npm i
       - run: npm run lint
       - run: npm run coverage
-      - name: Upload coverage to Coveralls
-        uses: coverallsapp/github-action@master
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-        if: ${{ matrix.node-version == 12 }}
+     - name: Upload coverage to Coveralls
+       uses: coverallsapp/github-action@master
+       with:
+         github-token: ${{ secrets.GITHUB_TOKEN }}
+       if: ${{ matrix.node-version == 14 }}
```

Comment:
We have actually switched over to Codecov quite some time ago on all the other libraries (so: all monorepo) and abanded Coveralls, this might be a likely reason why CI is not starting here, so in doubt just remove again, the we can at least see CI/tests going through and look for coverage results later on.

(or if you spot that it's easy to integrate/update you can of course also directly do)

Reply:
I went ahead and removed the coveralls step so we can get the tests running again.  It'd be nice to merge this if everything passes.